shell/common/engine.cc

```diff
@@ -487,6 +487,8 @@ void Engine::ConfigureAssetBundle(const std::string& path) {
   if (S_ISREG(stat_result.st_mode)) {
     asset_store_ = fxl::MakeRefCounted<blink::ZipAssetStore>(
         blink::GetUnzipperProviderForPath(path));
+    directory_asset_bundle_ = std::make_unique<blink::DirectoryAssetBundle>(
```

Comment:
If the path is a directory, the asset bundle will be automatically setup correctly and this method will return on line 484. I suggest you modify the invocation of the engine to pass the directory path instead of the FLX path. In the future, we can get rid of the `S_ISREG` check altogether.

The way this is currently written, this reads "read the asset bundle from this directory or FLX, and, if it is an FLX, also configure the asset bundle with the directory containing the FLX". This is unnecessarily complicated IMO.

Reply:
Thanks, and overall I agree. However, this is a small intermediate step towards removing the FLX, and right now the FLX is still used so we cannot return without setting the asset_store (as in line 488). Later, this will of course be rewritten.

Reply:
sgtm